common/server_game.cpp

```diff
@@ -156,12 +154,15 @@ void Server_Game::storeGameInformation()
     while (allUsersIterator.hasNext()) {
         Server_AbstractUserInterface *userHandler = server->findUser(allUsersIterator.next());
         if (userHandler)
-            userHandler->sendProtocolItem(*sessionEvent);
+            if (server->getStoreReplaysEnabled())
```

Comment:
I know you were going to suggest that.  The reason I didnt was due to me working on adding additional code in that block area that I ended up backing out.